corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -76,7 +76,7 @@
       queries or to identify unclean data.<br />
       This is a Feature Preview.
     {% endblocktrans %}
-    <a href="https://confluence.dimagi.com/display/commcarepublic/Feature+Previews"
+    <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143947412/Add-Ons"
```

Comment:
Just want to make sure this is the right link? I wouldn't be surprised if we changed 'feature preview' to 'add-on', but it's awkward that the translated text above still says "This is a Feature Preview". If we've changed the term, I'd imagine we'd also want to update the translation block.